plugin_tests/utils_test.py

```diff
@@ -181,7 +181,7 @@ def test_extract_orcid_id(self):
               "L1PCc6TzAG0_FnWqOcpzsl1bNrRNOFkgyg"
 
         res = extract_user_id(jwt)
-        self.assertEqual(res, "http://orcid.org/0000-0002-1756-2128")
+        self.assertEqual(res, "https://orcid.org/0000-0002-1756-2128")
```

Comment:
Does this test fail? DataONE Subjects when ORCID is used as the login method are not `https`.

Reply:
Ah, okay.